packages/playground/public/info/nextcloud.md

```diff
@@ -0,0 +1,5 @@
+---
+title: NextCloud
+---
+
+Nextcloud is a suite of client-server software for creating and using file hosting services. Nextcloud provides functionality similar to Dropbox, Office 365 or Google Drive. [Read the documentation](https://www.manual.grid.tf/terraform/advanced/terraform_nextcloud_aio.html) for more info.
```

Comment:
The URL should be changed to a playground one instead of terraform.

Reply:
Noted. New documentation [here](https://github.com/threefoldtech/info_grid/pull/240).